applications/games/modules/module.c

```diff
@@ -0,0 +1,1957 @@
+	if (self.menu_type == MT_IMAGE_TEXT_64_5X2)
+	{
+
+		TSU_RectangleSetSize(self.item_selector, 310, self.menu_option.image_size + 8);
+		init_position_x = 5;
+		init_position_y = 33;
+		selector_translate.z = ML_CURSOR;
+	}
+	else if (self.menu_type == MT_IMAGE_128_4X3)
+	{
+		TSU_RectangleSetSize(self.item_selector, self.menu_option.image_size + 4, self.menu_option.image_size + 4);
+		init_position_x = 9;
+		init_position_y = 41;
+		selector_translate.z = ML_CURSOR;		
+	}
+	else
+	{
+		TSU_RectangleSetSize(self.item_selector, self.menu_option.image_size + 4, self.menu_option.image_size + 4);
+		init_position_x = 8;
+		init_position_y = 40;
+		selector_translate.z = ML_CURSOR;
+	}
+
+	selector_translate.x = self.menu_cursel / self.menu_option.size_items_column * self.menu_option.padding_x + init_position_x;
+	selector_translate.y = ((self.menu_cursel + 1) - self.menu_option.size_items_column * (self.menu_cursel / self.menu_option.size_items_column)) * (self.menu_option.padding_y + self.menu_option.image_size) + init_position_y;
+
+	self.item_selector_animation = (Animation *)TSU_LogXYMoverCreate(selector_translate.x, selector_translate.y);
+	TSU_DrawableAnimAdd((Drawable *)self.item_selector, (Animation *)self.item_selector_animation);
+}
+
+static bool IsValidImage(const char *image_file)
+{
+	bool isValid = false;
+
+	ImageDimensionStruct *image = GetImageDimension(image_file);
+	if (image != NULL)
+	{
+		isValid = (image->height == 64 && image->width == 64) 
+			|| (image->height == 128 && image->width == 128) 
+			|| (image->height == 256 && image->width == 256);
+
+		free(image);
+	}
+
+	return isValid;
+}
+
+static int AppCompareGames(const void *a, const void *b)
+{
+	const GameItemStruct *left = (const GameItemStruct *)a;
+	const GameItemStruct *right = (const GameItemStruct *)b;
+	int cmp = 0;
+
+	char *left_compare = malloc(MAX_SIZE_GAME_NAME);
+	char *right_compare = malloc(MAX_SIZE_GAME_NAME);
+
+	memset(left_compare, 0, MAX_SIZE_GAME_NAME);
+	memset(right_compare, 0, MAX_SIZE_GAME_NAME);
+
+	if (left->is_folder_name && right->is_folder_name)
+	{
+		strcpy(left_compare, GetLastPart(left->folder, '/', 2));
+		strcpy(right_compare, GetLastPart(right->folder, '/', 2));
+
+		cmp = strcmp(left_compare, right_compare);
+	}
+	else if (left->is_folder_name)
+	{
+		strcpy(left_compare, GetLastPart(left->folder, '/', 2));
+		strncpy(right_compare, right->game, strlen(right->game) - 4);
+
+		cmp = strcmp(left_compare, right_compare);
+	}
+	else if (right->is_folder_name)
+	{
+		strncpy(left_compare, left->game, strlen(left->game) - 4);
+		strcpy(right_compare, GetLastPart(right->folder, '/', 2));
+		
+		cmp = strcmp(left_compare, right_compare);
+	}
+	else
+	{
+		strncpy(left_compare, left->game, strlen(left->game) - 4);
+		strncpy(right_compare, right->game, strlen(right->game) - 4);
+
+		cmp = strcmp(left_compare, right_compare);
+	}
+
+	free(left_compare);
+	free(right_compare);
+
+	return cmp > 0 ? cmp : -1;
+}
+
+// static int TotalPages()
```

Comment:
Do not push commented code plz.

Reply:
I didn't see that I left commented code, can I delete it, do I make the adjustment and upload it? Or what should I do?

Reply:
> I didn't see that I left commented code, can I delete it, do I make the adjustment and upload it? Or what should I do?

Yes, of course, continue to push to your repository in the same way and the changes will be automatic here too. This is the power of Github.

Reply:
Hi, I already uploaded them, did you get the modified files?

Reply:
Yes! But you forgot to fix type in    root Makefile.